framework/helpers/BaseInflector.php

```diff
@@ -566,8 +566,11 @@ public static function ordinalize($number)
      * @return string the generated sentence
      * @since 2.0.1
      */
-    public static function sentence(array $words, $twoWordsConnector = ' and ', $lastWordConnector = null, $connector = ', ')
+    public static function sentence(array $words, $twoWordsConnector = null, $lastWordConnector = null, $connector = ', ')
     {
+        if ($twoWordsConnector === null) {
+            $twoWordsConnector = ' ' . Yii::t('yii', 'and') . ' ';
```

Comment:
I'd rather put spaces inside the translation to allow changing using dictionary.

```php
$twoWordsConnector = Yii::t('yii', ' and ');
```